build.sbt

```diff
@@ -307,6 +308,22 @@ lazy val `kyo-cache` =
         )
         .jvmSettings(mimaCheck(false))
 
+lazy val `kyo-reactive-streams` =
+    crossProject(JVMPlatform)
+        .withoutSuffixFor(JVMPlatform)
+        .crossType(CrossType.Full)
+        .in(file("kyo-reactive-streams"))
+        .dependsOn(`kyo-core`)
+        .settings(
+            `kyo-settings`,
+            libraryDependencies ++= Seq(
+                "org.reactivestreams" % "reactive-streams" % "1.0.4",
+                "org.reactivestreams" % "reactive-streams-tck" % "1.0.4" % Test,
+                "org.scalatestplus" %% "testng-7-5" % "3.2.17.0" % Test
```

Comment:
Is this required to reuse the tck tests? Would it be possible to use scalatest like the other modules?

Reply:
We'll need to reimplement those test in scalatest though.